tests/integration_tests/modules/test_apt.py

```diff
@@ -152,7 +152,7 @@ def test_ppa_source(self, class_client: IntegrationInstance):
             "/etc/apt/sources.list.d/"
             "simplestreams-dev-ubuntu-trunk-{}.list".format(release)
         )
-        host = "launchpad" if release == "jammy" else "launchpadcontent"
+        host = "launchpadcontent" if release == "jammy" else "launchpad"
```

Comment:
LGTM, but this is going to fail again Ubuntu KK. What about this diff?
```
-        host = "launchpad" if release == "jammy" else "launchpadcontent"
         assert (
-            f"://ppa.{host}.net/simplestreams-dev/trunk/ubuntu"
+            f"://ppa.launchpad.net/simplestreams-dev/trunk/ubuntu"
+            in ppa_path_contents
+            or f"://ppa.launchpadcontent.net/simplestreams-dev/trunk/ubuntu"
             in ppa_path_contents
         )
```

Reply:
Or we can check `if release >= "jammy"` to make the fix more future proof.

Reply:
> ... What about this diff?
> 
> ```
> -        host = "launchpad" if release == "jammy" else "launchpadcontent"
>          assert (
> -            f"://ppa.{host}.net/simplestreams-dev/trunk/ubuntu"
> +            f"://ppa.launchpad.net/simplestreams-dev/trunk/ubuntu"
> +            in ppa_path_contents
> +            or f"://ppa.launchpadcontent.net/simplestreams-dev/trunk/ubuntu"
>              in ppa_path_contents
>          )
> ```

I'm going with this. Since `release` is just a string of the release name, I think this is easier than your next suggestion.

Reply:
Still it would work until ZZ as release names are alphabetically ordered, but maybe the `or` is just simpler.